google-home-community.groovy

```diff
@@ -562,6 +578,127 @@ private deviceTraitPreferences_Dock(deviceTrait) {
     }
 }
 
+@SuppressWarnings('UnusedPrivateMethod')
```

Comment:
You can change this to
```
@SuppressWarnings(['UnusedPrivateMethod', 'MethodSize'])
```
to silence the warning below.  I'm fine with suppressing that warning, since these settings methods aren't really something that can be easily broken down.

Reply:
Thanks, I knew I missing something.  Fixed.